collector/compile-benchmarks/README.md

```diff
@@ -206,6 +206,13 @@ Rust code being written today.
       --id Test --profiles=Check --scenarios=IncrPatched
       --include=$NEW_BENCHMARK`
   - Add the new entry to `collector/compile-benchmarks/README.md`.
+  - Add a new licensing entry to `collector/compile-benchmarks/REUSE.toml` (see existing entries
+  for inspiration).
+    - If the benchmark is artificial, use the `MIT` license and set Rust developers as the
+    copyright owners (see e.g. `await-call-tree` as an example).
```

Comment:
I didn't know if you'd want to ask for new benchmarks to be licensed the "normal" for rustc `MIT or Apache-2.0`?

Reply:
That's a very good point! And we can do that now, thanks to granular REUSE licenses. Thanks for the suggestion, updated the instructions.

Reply:
does the `await-call-tree` example still make sense?

Reply:
Yes, because the "Rust Project" developers is quite long, I didn't want to include the whole thing here. It should be copy-pasted from an existing example.

Reply:
yes but that example is only under the `MIT` license so I didn't know if it made more sense to point to a `MIT or Apache-2.0` licensed example (though I also understand pointing at the first rust-project example in the file for just the text)

Reply:
Yeah, now that example is only relevant because of the copyright, not the license. But I think that should be fine.